src/main/resources/static/events-feedback.js

```diff
@@ -38,9 +38,9 @@ function feedbackNext() {
     $.ajax({
         url: baseApp + "/submission/update",
         type: "POST",
-        data: {step: "overview"},
+        data: {step: "odm-upload"},
         success: function () {
-            window.location.href = baseApp + "/views/final";
+            window.location.href = baseApp + "/views/odm-upload";
```

Comment:
for the views to work, we need to have an odm-upload.html template in the folder /resources/templates, and this html templates in turn includes the odm-upload.js for interactions and ajax calls to work